client/src/components/chat.tsx

```diff
@@ -221,6 +221,7 @@ function Chat(props: { height: number }): JSX.Element {
   const [chatData, setChatData] = useState<ChatData>({
     messages: [],
     showAllAnswers: true,
+    lastChatAnswerId: 0,
```

Comment:
this is a data anomaly. The last chat id can be determined always by inspecting messages. Delete